File: food.py
from pygame import Surface, draw
from constants import GREEN, HEIGH, WIDTH
import random


class Food:
    def __init__(self) -> None:
        self.size = 10
        self.x = random.randint(0, ((WIDTH-self.size)/10)) * 10
        self.y = random.randint(0, ((HEIGH-self.size)/10)) * 10

    def update(self, screen: Surface = None, draw: bool = True):
        if screen != None and draw:
            draw.rect(screen, GREEN, [self.x,
                                      self.y, self.size, self.size])

    def pos(self) -> set[int, int]:
        return self.x, self.y

File: food.js
class Food {
  constructor() {
    this.shize = 20;
    this.updateLocation();
  }

  updateLocation() {
    let cols = floor(width / this.size);
    let rows = floor(height / this.size);

    this.pos = createVector(random(rows) * this.size, random(cols) * this.size);
  }

  show() {
    fill(0, 220, 0);
    rect(this.pos.x, this.pos.y, this.size, this.size);
  }
}
